client/my-sites/stats/stats-upsell/index.tsx

```diff
@@ -58,93 +58,80 @@ export default function StatsUpsell( { siteId }: { siteId: number } ) {
+									{
+										components: { b: <b /> },
+										args: {
+											planPrice: formatCurrency(
+												pricing.discountedPrice.monthly ?? pricing.originalPrice.monthly ?? 0,
+												pricing.currencyCode ?? '',
+												{
+													stripZeros: true,
+													isSmallestUnit: true,
+												}
+											),
+										},
+									}
+							  ) }
 					</div>
 					<div className="stats-upsell__features">
 						<div className="stats-upsell__feature">
 							<Gridicon icon="checkmark" size={ 18 } />
 							<div className="stats-upsell__feature-text">
-								{ translate(
-									'All stats available: traffic trends, sources, optimal time to post…'
-								) }
+								{ translate( 'View trends and data from any time period you choose' ) }
 							</div>
 						</div>
 						<div className="stats-upsell__feature">
 							<Gridicon icon="checkmark" size={ 18 } />
 							<div className="stats-upsell__feature-text">
-								{ translate( 'Download data as CSV' ) }
+								{ translate( 'Understand how visitors find and use your site' ) }
 							</div>
 						</div>
 						<div className="stats-upsell__feature">
 							<Gridicon icon="checkmark" size={ 18 } />
 							<div className="stats-upsell__feature-text">
-								{ translate( 'Instant access to upcoming features' ) }
+								{ translate( 'Track which posts and pages are most popular' ) }
 							</div>
 						</div>
 						<div className="stats-upsell__feature">
 							<Gridicon icon="checkmark" size={ 18 } />
 							<div className="stats-upsell__feature-text">
-								{ translate( '14-day money-back guarantee' ) }
+								{ translate( 'See where your visitors come from worldwide' ) }
 							</div>
 						</div>
 						<div className="stats-upsell__feature">
 							<Gridicon icon="checkmark" size={ 18 } />
 							<div className="stats-upsell__feature-text">
-								{ translate( 'All %(planName)s plan features', {
-									args: { planName: plan?.productNameShort ?? '' },
-								} ) }
+								{ translate( 'Monitor email engagement and downloads' ) }
+							</div>
+						</div>
+						<div className="stats-upsell__feature">
+							<Gridicon icon="checkmark" size={ 18 } />
+							<div className="stats-upsell__feature-text">
+								{ translate( 'Keep your data private and GDPR-compliant' ) }
 							</div>
 						</div>
 					</div>
+					<Button
+						variant="primary"
+						className="stats-upsell__button"
+						onClick={ onClick }
+						disabled={ isLoading }
+					>
+						{ ! plan?.productNameShort
+							? translate( 'Upgrade plan' )
+							: translate( 'Upgrade to %(planName)s', {
+									args: { planName: plan.productNameShort },
+							  } ) }
+					</Button>
+				</div>
+				<div className="stats-upsell__right">
+					<img src="/calypso/images/jetpack/stats-features.png?v=20241202" alt="Features" />
```

Comment:
Assuming this needs a fix to work on odyssey stats?

Reply:
Moved to stats assets folder, loads in odyssey now: https://sitetest603.wordpress.com/wp-admin/admin.php?flags=stats%2Fpaid-wpcom-v3&page=stats#!/stats/day/sitetest603.wordpress.com?flags=stats%2Fpaid-wpcom-v3&page=stats